lib/services/auth.dart

```diff
@@ -22,12 +23,26 @@ class AuthService {
     }
   }
 
+  Future<bool> checkVerification(FirebaseUser user) async {
+    if (user.isEmailVerified) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
   // sign up with email pass
 
-  Future<void> registerWithEmailAndPassword(
-      String email, String password) async {
-    var result = await _auth.createUserWithEmailAndPassword(
+  Future<void> registerWithEmailAndPassword(String email, String password,
+      String name, String mobilenum, String hostel, String sex) async {
```

Comment:
This has too many arguments for it to use positional parameters. Convert them to named parameters. Change the call appropriately

Reply:
fixed